### PR TITLE
docs: blogpost kratos accountability 

### PR DESCRIPTION
**Pull request by vinckr**
I fixed the formatting and added the header.

I will look into the contents more, maybe we can talk tomorrow more about it.

### CODE REVIEW

**Comment by tacurran** [comment]
@vinckr LGTM

File: src/pages/blog/account-recovery-password-reset-rethinking-user-accountability.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+
+Ory Kratos
+
+by Pluto Allegory
+
+With the release of Ory Kratos last month, we want to spend some time addressing
+some of the basics, the background and the overall approach. Hopefully, we can
+also help the community with this information.
+
+###
```
Comment (aeneasr) [inline src/pages/blog/account-recovery-password-reset-rethinking-user-accountability.md:33]: Headline missing :)

File: src/pages/blog/account-recovery-password-reset-rethinking-user-accountability.md [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+
+overline: |
+  Account Recovery and Password Reset: Rethinking User Accountability
+---
+
+## Account Recovery and Password Reset: Rethinking User Accountability
+
+Ory Kratos
+
+by Pluto Allegory
```
Comment (aeneasr) [inline src/pages/blog/account-recovery-password-reset-rethinking-user-accountability.md:27]: This is a bit confusing - is it a subtitle?

File: src/pages/blog/account-recovery-password-reset-rethinking-user-accountability.md [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+    account, password, security
+
+publishedAt: '2020-12-02'
+author: tacurran
+
+overline: |
+  Account Recovery and Password Reset: Rethinking User Accountability
+---
+
+## Account Recovery and Password Reset: Rethinking User Accountability
```
Comment (aeneasr) [inline src/pages/blog/account-recovery-password-reset-rethinking-user-accountability.md:23]: No need for this line, it is already in the header!

File: src/pages/blog/account-recovery-password-reset-rethinking-user-accountability.md [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+with the "GNU Manifesto," and Microsoft released Windows 1.0. There was a famous
+Super Bowl commercial featuring lemmings, and towards the end of the year Steve
+Jobs left Apple Computer. Unix was one of the dominant operating systems at the
+time. In April, 1985 the Unites States Department of Defense published, as part
+of the
+[Rainbow Series](https://web.archive.org/web/20110720190716/http://iaarchive.fi/rainbow.html)
+of books outlining and explaining computer security standards,the Password
+Management Guideline otherwise known as the "Green Book" [CSC-STD-002-85,
+Library No.
+S-226,994][national institute of standards and technology https://fas.org/irp/nsa/rainbow/std002.htm]
```
Comment (aeneasr) [inline src/pages/blog/account-recovery-password-reset-rethinking-user-accountability.md:46]: This markdown link is incorrect

File: src/pages/blog/account-recovery-password-reset-rethinking-user-accountability.md [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+due the green colour of its cover. The Green Book made reference, as did every
+other book in the Rainbow Series, to the "Orange Book" [CSC-STD-001-83], or
```
Comment (aeneasr) [inline src/pages/blog/account-recovery-password-reset-rethinking-user-accountability.md:48]: This looks like a link but the URL is missing

File: src/pages/blog/account-recovery-password-reset-rethinking-user-accountability.md [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+Book@ that looks into the Orange Book|s legacy and meaning in some of today's
+standards.
+
+## Conclusion
+
+What is accountability? Has has the evolution of the Internet validates the
+earlz concepts while adding more intelligence into the network so accountability
+can be shared between man and machine.
+
+[DoD Computer Security Center. Department of Defense Trusted Computer System
```
Comment (aeneasr) [inline src/pages/blog/account-recovery-password-reset-rethinking-user-accountability.md:217]: Looks like a link but URL is missing

File: src/pages/blog/account-recovery-password-reset-rethinking-user-accountability.md [hunk 7]
```diff
@@ -0,0 +1,222 @@
...
+Evaluation Criteria, CSC-STD-00183, 15 August 1983.]
+
+S. B. Lipner, "The Birth and Death of the Orange Book," in IEEE Annals of the
```
Comment (aeneasr) [inline src/pages/blog/account-recovery-password-reset-rethinking-user-accountability.md:220]: Maybe format the source as a list?

Comment (tacurran) [reply src/pages/blog/account-recovery-password-reset-rethinking-user-accountability.md:219]: @vinckr  lets decide on a convention for this.  I think lists in md will work fine

**Pull request merged**